modules/skybrowser/src/wwtdatahandler.cpp

```diff
@@ -330,23 +331,23 @@ void WwtDataHandler::loadImages(const std::string& root,
         }
     }
 
-    // Sort images in alphabetical order
```

Comment:
Why no more sorting?  I think if it doesn't cost too much, timewise it'd be nicer to keep it

Reply:
I can't remember 😅 I'll implement the sort